macros/DekiScript-Uri.ejs

```diff
@@ -1,32 +0,0 @@
-<% module.exports = buildAPI({
- 
-    // Decompose a URI into its component parts. (Obsoletes uri.parse)
-    // [See also](http://developer.mindtouch.com/en/docs/DekiScript/Reference/DekiScript_Functions_and_Variables/Uri.Parts)
-    parts: function (str) {
```

Comment:
Nice catch noticing that this isn't used and not including it your new code! Well, I should say it isn't used by anything that is actually executed, because it appears to be used within `kumascript/tests/fixtures/server/macros/xulelem.ejs` but that is never executed and seems to be only used as a filename in the list of macros returned for testing purposes. I suppose we could remove `kumascript/tests/fixtures/server/macros/xulelem.ejs` (and update `kumascript/tests/fixtures/server/macros-expected.json` accordingly)?

Reply:
I'll dig into that and try to clean it up. It appears with different capitalizations in other fixtures, as well.

Reply:
Resolved in 5b40b6e
I deleted the xulelem.ejs and two other macros in that fixtures directory that were not being used by tests except for the /macros/ endpoint test. Those were the three files that used this old Uri.Parts() function.